<?xml version="1.0"?>
<launch>
  <!-- A* Planner Node -->
  <node pkg="lenna_astar" type="astar_node" name="astar_planner" output="screen">
    <!-- Add any parameters here -->
  </node>
</launch>
